noxfile.py

```diff
@@ -98,7 +104,7 @@ def run_unit(session):
     set_environment_variables(PYBAMM_ENV, session=session)
     session.install("setuptools", silent=False)
     session.install("-e", ".[all,dev,jax]", silent=False)
-    session.run("python", "run-tests.py", "--unit")
+    session.run("python", "-m", "pytest", "-svra", "-m", "unit")
```

Comment:
I know I suggested adding this earlier, but I think we have something better. Let's add these to `addopts` in `pyproject.toml` under `[tool.pytest.ini_options]` instead.
```suggestion
    session.run("python", "-m", "pytest", "-m", "unit")
```

Reply:
Also, can we make them run with `pytest --unit` instead of `pytest -m unit`? No big deal if we can't, just wanted to know if it's possible to do so with how you have configured the markers